astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1133,6 +1138,8 @@ def __init__(
         :type: list(NodeNG)
         """
 
+        self.instance_attrs = {}
```

Comment:
```suggestion
        self.instance_attrs: Dict[str, List[nodes.NodeNG]] = {}
```

Perhaps we can narrow it down even more, but for now this is enough I think?